EventHandlers/PrebidMobileAdMobAdapters/Sources/PrebidAdMobNativeAdapter.swift

```diff
@@ -45,6 +45,15 @@ public class PrebidAdMobNativeAdapter:
     
     public var store: String?
     
+    public var privacyUrl: String? {
```

Comment:
This property doesn't make any sense, because it is absent in `GADMediatedUnifiedNativeAd`. So the GMA SDK won't use it. 

The sense of the request in the referenced ticket was just about adding the property to `NativeAd` class. 

```
 public var privacyUrl: String? {
        set {
            nativeAdMarkup?.privacy = newValue
        }
        get {
            nativeAdMarkup?.privacy
        }
    }
```

So publishers can use `nativeAd.privacyUrl` instead of `nativeAd.nativeAdMarkup?.privacy`.